lib/ability.rb

```diff
@@ -148,7 +148,7 @@ def initialize(user, resource, controller = nil)
 
         can :index, User if controller.is_a?(FacilityUserReservationsController)
 
-        can [:list, :show], Facility
+        can [:list, :dashboard, :show], Facility
```

Comment:
@jhanggi, I just wanted to double check--I added this to the ability file to resolve the sanger spec failures. This permission is correctly located, right? It shouldn't be overreaching anywhere?